### PR TITLE
Add cluster/fleet command reloading

### LINKED ISSUE #860: ,reload doesn't work across clusters

### PR DESCRIPTION
**Pull request by Brainicism**
Closes #860

### CODE REVIEW

File: src/commands/admin/reload.ts [hunk 1]
```diff
@@ -1,30 +1,53 @@
...
 
 const logger = new IPCLogger("reload");
 
-export default class EvalCommand implements BaseCommand {
-    call = async ({ message }: CommandArgs) => {
+export enum ReloadType {
+    CLUSTER = "cluster",
+    ALL = "all",
+}
+export default class ReloadCommand implements BaseCommand {
```
Comment (sabihismail) [inline src/commands/admin/reload.ts:15]: new line between objects or ur cringe fr

**Review by sabihismail — COMMENTED** [review]
yea fr do this or ur cringe

File: src/commands/admin/reload.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ export enum ReloadType {
     CLUSTER = "cluster",
     ALL = "all",
 }
+
```
Comment (sabihismail) [inline src/commands/admin/reload.ts:15]: ok we gamin

**Comment by sabihismail** [comment]
ok now we gamin

File: src/commands/admin/reload.ts [hunk 3]
```diff
@@ -1,30 +1,54 @@
...
+    ALL = "all",
+}
+
+export default class ReloadCommand implements BaseCommand {
+    validations = {
+        minArgCount: 1,
+        maxArgCount: 1,
+        arguments: [
+            {
+                name: "seekType",
```
Comment (taahamahdi) [inline src/commands/admin/reload.ts:22]: name reloadType

File: src/commands/admin/reload.ts [hunk 4]
```diff
@@ -1,30 +1,54 @@
...
-            logger.error(`Error reloading KMQ commands: err = ${e}`);
+            logger.error("Error transpiling KMQ commands");
             sendErrorMessage(MessageContext.fromMessage(message), { title: "Error Reloading", description: `Uh oh.\n${e}` });
+            return;
         }
+
+        const reloadType = parsedMessage.components[0] as ReloadType;
+        if (reloadType === ReloadType.ALL) {
+            state.ipc.broadcast("reload_commands");
+            sendInfoMessage(MessageContext.fromMessage(message), { title: "Reloading all clusters", description: "See logs for completion status" });
```
Comment (taahamahdi) [inline src/commands/admin/reload.ts:47]: Nit: title capitalization between this and below info message varies

**Review by taahamahdi — CHANGES_REQUESTED** [review]

**Review by taahamahdi — APPROVED** [review]

**Pull request merged**